README.md

```diff
@@ -244,48 +93,22 @@ Feel free to check the [issues page](../../issues/).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- SUPPORT -->
-
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+If you like this project follow me on my social accounts to be notified of future projects
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+I would like to thank Microverse
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FAQ (optional) -->
-
-## ❓ FAQ <a name="faq"></a>
-
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Please 🙏🏻  add the `MIT` license properly because including the **MIT** *License* in your `GitHub` repository **README** is crucial for several reasons. Firstly, the **MIT** License is a permissive `open-source` license that grants users the freedom to use, modify, and distribute the software without legal restrictions. By choosing the *MIT* License 🗒️, you are encouraging collaboration, inviting other developers to contribute, and fostering a vibrant open-source community. Additionally, using the *MIT* License helps protect your work by clearly defining the terms and conditions under which it can be used, providing a level of legal 🚨 clarity and ensuring that proper attribution is given. Furthermore, the widespread popularity and familiarity of the *MIT* License make it easy for others to understand and comply with the licensing requirements, which ultimately promotes the adoption and usage of your software 💻 .

### Broken Link
![image](https://github.com/hamidazim321/vet-clinic-db/assets/66476329/5f597cdd-f773-4bf4-84b6-e6464e6bd7cf)

Reply:
Thanks for the review. I have added an MIT license